pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -666,6 +667,9 @@ def resolve_training_type_plugin(self, training_type: TrainingTypePlugin) -> Tra
             training_type.cluster_environment = self.cluster_environment
             self._cluster_environment = proxy(self.cluster_environment)
 
+        if hasattr(training_type, "checkpoint_plugin") and getattr(training_type, "checkpoint_plugin") is None:
+            training_type.checkpoint_plugin = TorchCheckpointIOPlugin()
+
```

Comment:
is this needed since they already have the property?

Reply:
I removed this and added the below, hope this is what you had in mind!

https://github.com/PyTorchLightning/pytorch-lightning/blob/feat/ckpt_plugin/pytorch_lightning/plugins/training_type/training_type_plugin.py#L43